herddb-core/src/main/java/herddb/sql/CalcitePlanner.java

```diff
@@ -323,7 +323,7 @@ public TranslatedQuery translate(String defaultTableSpace, String query, List<Ob
                         new SQLPlannedOperationStatement(values),
                         values
                 );
-                return new TranslatedQuery(executionPlan, new SQLStatementEvaluationContext(query, parameters, forceAcquireWriteLock));
+                return new TranslatedQuery(executionPlan, new SQLStatementEvaluationContext(query, parameters, false, false));
```

Comment:
Why this was forceAcquireWriteLock and now is always false?

Reply:
`false` is the initial value.
It is not the planner that decides this flag but we are setting it is ServerSideConnectionPeer